data/wanderer/wanderer news.txt

```diff
@@ -0,0 +1,127 @@
+
+news "wanderer civilians, no dialog"
+	location
+		government "Wanderer"
+		not attributes "evacuation"
+	name
+		word
+			"Gardener"
+	message
+		word
+			"You see a Wanderer carefully place some seeds in the ground. They then spread some sort of light grey powder over them."
+			"From the spaceport you can see some buildings with plants growing on their sides. A Wanderer prunes and waters them in mid-air."
+			"While walking through the spaceport, you see a large greenhouse full of various plants. A Wanderer inside is tending to them."
+			
+	name
+		word
+			"Group of Wanderers"
+	message
+		word
+			"While walking through the spaceport, you see several Wanderers flying through the air. You duck to avoid them."
```

Comment:
Are they flying or landing? Seems odd to duck if they're flying, especially since wanderers are implied to be rather taller than you are so nearby wanderers would have been even taller than you.

Reply:
I changed it, does it make more sense now?

Reply:
It does make more sense yes. Feel free to resolve.